src/config/languages.json

```diff
@@ -0,0 +1,44 @@
+{
+    "en": {
+        "slug": "en",
+        "uioSlug": "en",
+        "dir": "ltr",
```

Comment:
Do you still need this with the `rtl-detect`?

Reply:
I don't think it's a bad idea to include this data. For example, in a base template, it's easier to do this:

```nunjucks
<html lang="{{ locale }}" dir="{{ supportedLanguages[locale].dir }}">
```

Than somehow use that `rtl-detect` helper to figure out the direction of the current locale.